.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
 
 script:
   - pipenv run flake8 parselglossy
-  - pipenv run py.test -v --cov=parselglossy tests/ --doctest-module
+  - pipenv run py.test -s -v --cov=parselglossy tests/ --doctest-module
```

Comment:
:+1: for adding the flag. Don't forget to add it also to `.ci/azure-steps.yml`

Reply:
Done.